src/Build/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -763,6 +768,15 @@ public string OutputResultsCacheFile
             set => _outputResultsCacheFile = value;
         }
 
+        /// <summary>
+        /// Determines whether MSBuild will cache the results of builds after EndBuild to speed up future builds.
+        /// </summary>
+        public bool CacheBuildResults
```

Comment:
Some bikeshedding on the name.

`CacheBuildResults` sounds a bit generic. When I initially read the name I thought the PR would be about not using a ResultsCache at all, as opposed to just clearing it on EndBuild. Maybe `ResetBuildResultsCache` instead?